lib/App/DuckPAN.pm

```diff
@@ -79,6 +79,13 @@ has duckpan_packages => (
 	builder => 1,
 );
 
+has fathead_output => (
```

Comment:
This gives me a way to access the `output_txt` option elsewhere (e.g Web.pm), without having to pass along new args from /cmd/server.pm and /cmd/query.pm.

AFAICT the `option` needs to be defined in `server.pm` and `query.pm` and not `DuckPAN.pm` otherwise the command would be:

`duckpan -o /path/to/output.txt server` which IMO feels awkward.

There may be a better solution, but this worked ;)